index.html

```diff
@@ -983,7 +983,11 @@ <h4>Semantic search: SPARQL</h4>
                 <li>501 (Not Implemented): SPARQL API not supported.</li>
               </ul>
             </p>
-      			</section>
+	<span class="rfc2119-assertion" id="tdd-distributed-search-semantic">if implemented, the SPARQL API SHOULD implement the SPARQL 1.1 Federated Query standard [[sparql11-overview]]</span>
+	<p class="ednote" title="SSE Authorization Header">
+             The WoT Thing Directory SHOULD implement mechanisms to prevent DDoS attacks in the SPARQL search API
```

Comment:
Change to use two assertions:
1. A WoT Thing Description Directory MAY implement federation in its SPARQL query API.
2. If implemented, the SPARQL API MUST implement the SPARQL 1.1 Federated Query standard [[sparql11-overview]]

Also, use WoT Thing Description Directory (TDD to avoid confusion with "TD").

Resolved in the meeting 07.12.2020 to merge after these changes.

Reply:
Changes requested addressed

Reply:
Updates look good, but ed note is formatted as an assertion and has some other issues (title is wrong, for instance).  I'm going to merge and then edit the note...